indexes/plugin.js

```diff
@@ -108,7 +108,8 @@ module.exports = class Plugin {
      */
     this.forcedFlush = this._flush.bind(this, true)
 
-    const liveFlush = debounce(this.flush, 250)
+    const flushDebounce = (configDb2 && configDb2.flushDebounce) || 250
+    const liveFlush = debounce(this.flush, flushDebounce)
```

Comment:
Totally optional suggestion:

```suggestion
    const liveFlush = debounce(this.flush, configDb2?.flushDebounce ?? 250)
```

Reply:
skipping because node 12 will cry :D